src/views/Space/Overview.vue

```diff
@@ -24,7 +28,9 @@ const network = computed(() => getNetwork(props.space.network));
 const spaceStarred = computed(() => spacesStore.starredSpacesIds.includes(spaceIdComposite));
 const isController = computed(() => compareAddresses(props.space.controller, web3.value.account));
 
-const externalUrl = computed(() => sanitizeUrl(props.space.external_url));
+const externalUrl = computed(() =>
```

Comment:
Not sure if this was a bug, but I added the same checks as for the others like `twitterUrl` @Sekhmet

Reply:
Empty string would be handled by sanitizeUrl itself. We have this check for other URls because we don't pass value by itself but combine it into longer string (`https://twitter.com/${props.space.twitter}`) so even if props.space.twitter is empty we would still get valid URL (where we don't want anything).

Reply:
Ok reverted this